resume.md

```diff
@@ -1,49 +1,58 @@
-# FIRST LAST NAME
+# Tejesh Bhaumik
 
-(123) 456 - 7890 | email@gmail.com
+(123) 456 - 7890 | tejeshbhaumik@gmail.com
 
-www.website.com | github.com/myusername | linkedin.com/myusername
+| github.com/Tejesh2001 | linkedin.com/TejeshBhaumik
 
 ## EDUCATION
-**University of College** | B.S. Degree ... *MONTH 20XX*
-* **Coursework**: Data Structures, Algorithms, Upper Division CS Elective, Math Course, Statistics, Data Science, ETC.
+**University of College** | B.S. of Computer Science + Advertising... *May 2023*
+* **Coursework**: Data Structures, Algorithms, Advanced Linear Algebra, Statistics, Data Science, Probability, Brand Strategy.
 
 ## SKILLS
-* **Languages**: Python, Java, C, SQL, ...
-* **Libraries/Frameworks**: Pandas, TensorFlow, PyTorch, ...
-* **Version Control**: Git, Bitbucket, Sourcetree, GithubDesktop
+* **Languages**: Python, Java, C++, SQL, NodeJs, Android
+* **Libraries/Frameworks**: Pandas, Numpy, Scipy, Sympy
+* **Version Control**: Git, GithubDesktop
 * *...*
 
 ## EXPERIENCE
-**Software Engineer**: Company | City, State ... *Month 20XX - Present*
-* Implemented XYZ Microservice using TECHNOLOGY to delivery PRODUCT to 123 Customers Globally
-* Architected CORE INFRASTRUCTURE LIBRARY used across X teams to BLAH BLAH
-* I did X for Y to accomplish Z
-* ...
+**Android Developer**: Crowd Dynamics Lab   | Remote *September 2020 - Present*
+* Assist in the implementation of an Android tracer app using IoT principles
+
+
+**Backend Developer**: Ensemble Education | Remote *August 2020 - October 2020*
+* Implemented social media logins, file systems and assisted in building REST APIS in NodeJS and C++ 
```

Comment:
Which social media logins?  What APIs did you use?  What REST APIs did you make and why does this matter?  Who is the target audience of this?  What is the impact of this project?  Who does this help?